springboot-assignment/src/main/java/com/springboot/assignment/AssignmentApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.springboot.assignment;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class AssignmentApplication {
+
```

Comment:
@Bean
	public ModelMapper modelMapper() {
		return new ModelMapper();
	}
	
create a modelmapper this way and use this throughout